exporter/cassandraexporter/config.go

```diff
@@ -21,3 +22,8 @@ type Replication struct {
 type Compression struct {
 	Algorithm string `mapstructure:"algorithm"`
 }
+
+type Auth struct {
+	UserName string `mapstructure:"username"`
+	Password string `mapstructure:"password"`
```

Comment:
```suggestion
	Password configopaque.String `mapstructure:"password"`
```
Passwords and sensitive information need to use the configopaque module. Context in #17273. There are a lot of examples there too on how to use it instead of a plain string.